tests/unit/test-loader-test.js

```diff
@@ -3,7 +3,11 @@ import { module, test } from 'qunit';
 
 module('Unit | test loader', {
   beforeEach() {
-    this.TestLoader = window.require('ember-cli/test-loader')['default'];
+    try {
+      this.TestLoader = window.require('ember-cli-test-loader/test-support/index')['default'];
```

Comment:
> we have `require.entries` which can be used for existence checks.

Ah, hadn't thought of that. I'll update the code to take that approach instead.

> it is also somewhat strange that this is coupled to window at all, isn't just ambient require sufficient?

Yeah, it is. I was just following the code that was already present.